src/adapters/mc/asb/AzureLinuxBaseline.mof

```diff
@@ -1222,6 +1301,7 @@ instance of OsConfigResource as $OsConfigResource78ref
 instance of OsConfigResource as $OsConfigResource79ref
 {
     ResourceID = "Ensure all bootloaders have password protection enabled";
```

Comment:
Is it OK that the ResourceID is not mapping exactly with the corresponding Name ("All bootloaders should have password protection enabled" in this instance)? Is that agreed?

Reply:
All names in here follow a similar consistent pattern. There are some slight changes from v1 etc. Names will be one last time reviewed before release.